x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateTimeParseProcessorTests.java

```diff
@@ -130,5 +188,18 @@ public void testParsing() {
                 .asProcessor()
                 .process(null)
         );
+        // TimeParse
+        assertEquals(
```

Comment:
Please change test tests similarly to https://github.com/elastic/elasticsearch/pull/55223/files#diff-4469bf63c6bd559fd4d04c24e52f23e4R172 where the session timezone is used for the resulting time.